content/ekscloudwatchcontainerinsights/cwcinstall.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Installing Container Insights"
+chapter: false
+weight: 5
+---
+
+
+<h4>Installing CloudWatch Container Insights using QuickStart: </h4>
+
+We'll be using the QuickStart to make the install simple and easy for the Container Insights. 
+
+You can find the full information and manual install steps here: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Container-Insights-setup-EKS-quickstart.html 
+
+
+From your Cloud9 Terminal you will just need to run the following command:
+
+
+```
+curl https://raw.githubusercontent.com/aws-samples/amazon-cloudwatch-container-insights/master/k8s-yaml-templates/quickstart/cwagent-fluentd-quickstart.yaml | sed "s/{{cluster_name}}/eksworkshop-eksctl/;s/{{region_name}}/us-east-2/" | kubectl apply -f -
+```
+
+With this quick start it will push the necessary daemon sets to collect the data for CloudWatch Containers Insights.
+
+
+<img src="/images/ekscwci/cwdaemon.png">
```

Comment:
We shouldn't need the embedded html... can you swap to markdown format, and also move images under /static/cwci/ (or some other relevant directory name)?